source/codegen/templates/library.h.mako

```diff
@@ -43,8 +44,14 @@ class ${service_class_prefix}Library : public ${namespace_prefix}::${service_cla
   f = functions[method_name]
   parameters = f['parameters']
   return_type = f['returns']
+  c_name = service_helpers.get_cname(functions, method_name, c_function_prefix)
 %>\
+## "Private" methods aren't in the public header, so generate the params from the metadata.
+% if service_helpers.is_private_method(f):
```

Comment:
Note: Context. There's currently [one private method](https://github.com/ni/grpc-device/blob/9b335069be8ec0b4f5fb7b76dfb28f600426cdc0/source/codegen/metadata/nidcpower/functions.py#L2282) in grpc-device. There's a similar special case (for the same reason) [in library.cpp.mako](https://github.com/ni/grpc-device/blob/ba49f0d3fd6ad33bf8e3d6c7c4c2b595f37df08e/source/codegen/templates/library.cpp.mako#L70).

@gregstoll we could consider pulling out the `expand_varargs=False` condition/support. This isn't adding a ton of dead code, though.

Reply:
Yeah, once this goes in I can clean up the `expand_varargs` stuff